fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Headers.kt

```diff
@@ -1,5 +1,8 @@
 package com.github.kittinunf.fuel.core
 
+import java.util.HashMap
+import java.util.HashSet
```

Comment:
Why do we need this? We didn't need this before??

Reply:
Without this, the IDE shows red though .... and auto adds it for me.

Reply:
Fixed sir! a069508

Reply:
Weird?? Doesn't show that for me 🗡